_layouts/home.html

```diff
@@ -1,5 +1,64 @@
 ---
-layout: default
+layout: table_wrappers
 ---
 
-{{ content }}
+<!DOCTYPE html>
```

Comment:
Some of this looks like repeated code that should be shared between default and home. If you're trying to split them up to avoid, say showing the sidebar on the home page, then both home and default should inherit from another page to avoid repeating stuff.

Reply:
I see your point and we can figure out making the code cleaner after having the basic implementation. I'm just trying to see if it works with our site code right now.